docs/packaging.md

```diff
@@ -1,12 +1,12 @@
 # Packaging
 
-The Identity Service (IS), Keys Service (KS) and Certificates Service (CS) have been explicitly designed to not be Azure IoT Edge-specific; they can be used on non-Edge Azure IoT devices also. To this end, we want to cleanly separate these components from existing association with IoT Edge in the form of hosting it in a separate source repository and shipping it as a separate package. Furthermore, the existing IoT Edge daemon will be modified to depend on these components for provisioning the device, managing module identities, and managing cryptographic keys and certificates, and thus its only responsibility will be to act as a Module Runtime (MR).
+The Identity Service (IS), Keys Service (KS) and Certificates Service (CS) have been explicitly designed to not be Azure IoT Edge-specific; they can be used on non-Edge Azure IoT devices also. To this end, we want to cleanly separate these components from existing association with IoT Edge in the form of hosting it in a separate source repository and shipping it as a separate package. Furthermore, the existing IoT Edge daemon (`iotedged`) will be modified to depend on these components for provisioning the device, managing module identities, and managing cryptographic keys and certificates. Its only responsibility will be to act as a Module Runtime (MR).
```

Comment:
It would be great to be more specific. How about below:

"Iotedged's only responsibility will be to act as a container Module Runtime. "

Reply:
How about, "The `iotedged`'s binary name will become `aziot-edged` and its only responsibility will be to act as a Module Runtime for containerized Edge modules."?

Reply:
I made a change to to briefly highlight that `iotedged` is renamed to `aziot-edged` in the prior sentence.  Then in this one I went with basically what you proposed:  `aziot-edged`'s only responsibility will be to act as a Module Runtime (MR) for containerized Edge modules.